content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -110,11 +129,100 @@ A ‘Jump to column’ feature is available for searching for specific metadata
+
+## Field Notes Functionality
+
+https://microbiomedata.org/field-notes/
+
+The NMDC Field Notes app is for mobile device use. It runs on both Apple (iOS) and Android devices. It's currently available for beta testing. See (TODO) to sign up!
+
+This app is designed to streamline real-time registration and collection of standardized metadata and field measurements. 
+
+Upon first use, each section of the app will automatically launch a quick tour. You can press the "→" button to continue the tour or press the "X" button to end it. To re-initialize the tours at any time, visit the Settings page and tap "Reset tours".
+
+The "Guide" section of the app contains recommendations about fieldwork in general, and about collecting metadata for specific sample type(s). We recommend viewing it before going out into the field.
+
+TODO: Add screen shot images
+
+The NMDC Field Notes app requires ORCiD authentication to access it. 
+
+Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi. The app is available for offline use, but requires internet connection to **start** a submission. 
+
+
+TODO: Update all these to field notes!
+
+(Step 1 of 3) Welcome to the NMDC Field Notes mobile app! 
+
+[![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)
+
+Once signed in with an ORCiD, you will have the ability to access existing studies or create a study using the 'New’ button. Details about each section are outlined below. Users can also return to existing submissions saved under the ORCiD account to resume their work at any point. 
+
+### New Study
+
+[![](../_static/images/howto_guides/portal_guide/study_info.png)](../_static/images/howto_guides/portal_guide/study_info.png)
+
+The Study Information page requires a valid ‘Study Name’ and a valid email address. All other fields are optional and can be completed in the app or in the Submission Portal. Study names and all other information can be updated or added via the app or Submission Portal. See [header](#submission-portal-functionality) [header](#study) for more details about Study information.
+
+Also in the New Study you will select the "Environment Template" that best fits your samples. The templates available correspond to the GSC Environmental Extensions. More information is available above in sections [header](#environmental-package). Multiple environments can be selected and submitted in one submission.
+
+Select "Save" once study information has been complete.
+
+### Select Fields
+
+TODO: Add Image
+
+The Field Notes App allows the submitter to select the metadata terms (TODO what word?!) that will be visible and available on the app and when working in the field. 
```

Comment:
@pkalita-lbl @eecavanna suggestions on the word here?
metadata fields
metadata terms 
?

Reply:
We use the term "fields" in the app itself.